<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlateInspector</class>
 <widget class="QWidget" name="PlateInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_18">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Viewing Page 0 of 0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_26">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_29">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Page Navigation</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_12">
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButton_18">
           <property name="toolTip">
            <string>Navigate to the next page of images</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
           <property name="shortcut">
            <string>Right</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pushButton_21">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Switch to the next imaging run for this sample</string>
           </property>
           <property name="text">
            <string>Next Date</string>
           </property>
           <property name="shortcut">
            <string>Up</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QPushButton" name="pushButton_22">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Switch to alternative spectrum for this sample</string>
           </property>
           <property name="text">
            <string>Swap Spectrum</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Up</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButton_20">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Switch to previous imaging run for this sample</string>
           </property>
           <property name="text">
            <string>Previous Date</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButton_17">
           <property name="toolTip">
            <string>Navigate to previous page of images</string>
           </property>
           <property name="text">
            <string>Previous</string>
           </property>
           <property name="shortcut">
            <string>Left</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Image Filtering</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="toolTip">
             <string>Highlight images that meet your selected filters</string>
            </property>
            <property name="title">
             <string>Classifier</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="checkBox_21">
               <property name="text">
                <string>Human</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBox_22">
               <property name="text">
                <string>MARCO</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Image Type</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBox_25">
               <property name="text">
                <string>Precipitate</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkBox_26">
               <property name="text">
                <string>Other</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBox_23">
               <property name="text">
                <string>Crystal</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkBox_24">
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QCheckBox" name="checkBox_6">
               <property name="text">
                <string>Favorite</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>Image Coloring</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBox_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>Color Intensity</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSlider" name="horizontalSlider">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="pageStep">
             <number>1</number>
            </property>
            <property name="sliderPosition">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Classifier</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>Crystal</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>MARCO</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Human</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_4">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QComboBox" name="comboBox_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>Precipitate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Labels</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>Human Classification</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Cocktail Number</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>MARCO Classification</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Include Checked Labels</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Well</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_9">
         <attribute name="title">
          <string>Plate View</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_29">
            <property name="toolTip">
             <string>Keep the aspect ratio of the images in the grid. Favors accuracy to the image over image size.</string>
            </property>
            <property name="text">
             <string>Preserve Aspect Ratio</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_7"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_28">
            <property name="toolTip">
             <string>Apply image colorings selected in the &quot;Image Coloring&quot; tab</string>
            </property>
            <property name="text">
             <string>Apply Image Colors</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_27">
            <property name="toolTip">
             <string>Apply image filters selected in the &quot;Image Filtering&quot; tab.</string>
            </property>
            <property name="text">
             <string>Apply Image Filters</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>Images Per Plate</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_19">
            <property name="toolTip">
             <string>Apply the currently selected coloring and filtering options if selected</string>
            </property>
            <property name="text">
             <string>Apply Plate Settings</string>
            </property>
            <property name="shortcut">
             <string>A</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_23">
            <property name="toolTip">
             <string>Reload the current view. Like a refresh button.</string>
            </property>
            <property name="text">
             <string>Reload Current View</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Save View</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="plateViewer" name="plateViewer" native="true">
     <property name="toolTip">
      <string>Plate views are shown here. If you are viewing an HWI Run each view is a subsection of the larger plate.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plateViewer</class>
   <extends>QWidget</extends>
   <header>polo/widgets/plate_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
